.gitignore

```diff
@@ -6,7 +6,7 @@
 .packages
 build/
 # If you're building an application, you may want to check-in your pubspec.lock
```

Comment:
So if `pubspec.lock` is being ignored, does this comment still make sense?  My understanding is that `pubspec.lock` gets generated locally based on `pubspec.yaml` and that not every generated `pubspec.lock` will be universally portable.

Reply:
You are right I will remove this comment.

Reply:
Fixed.